### PR TITLE
MIFOSX-1891 Improvements to extend term for loans following daily repaym...

### PR DESCRIPTION
**Pull request by chandrikamohith**
...ent schedule.

### CODE REVIEW

File: mifosng-provider/src/main/java/org/mifosplatform/organisation/workingdays/domain/WorkingDays.java [hunk 1]
```diff
@@ -65,6 +73,13 @@ public Integer getRepaymentReschedulingType() {
             actualChanges.put(repaymentRescheduleTypeParamName,  WorkingDaysEnumerations.workingDaysStatusType(newValue));
             this.repaymentReschedulingType = RepaymentRescheduleType.fromInt(newValue).getValue();
         }
+        
+        final String extendTermForDailyRepayments = "extendTermForDailyRepayments";
```
Comment (pramodn02) [inline mifosng-provider/src/main/java/org/mifosplatform/organisation/workingdays/domain/WorkingDays.java]: use constant from WorkingDaysApiConstants

File: mifosng-provider/src/main/java/org/mifosplatform/portfolio/loanaccount/loanschedule/domain/DefaultScheduledDateGenerator.java [hunk 2]
```diff
@@ -62,13 +63,22 @@ public LocalDate adjustRepaymentDate(final LocalDate dueRepaymentPeriodDate, fin
             final LocalDate nextDueRepaymentPeriodDate = getRepaymentPeriodDate(loanApplicationTerms.getRepaymentPeriodFrequencyType(),
                     loanApplicationTerms.getRepaymentEvery(), adjustedDate, loanApplicationTerms.getNthDay(),
                     loanApplicationTerms.getWeekDayType());
-            adjustedDate = WorkingDaysUtil.getOffSetDateIfNonWorkingDay(adjustedDate, nextDueRepaymentPeriodDate,
```
Comment (pramodn02) [inline mifosng-provider/src/main/java/org/mifosplatform/portfolio/loanaccount/loanschedule/domain/DefaultScheduledDateGenerator.java]: why this change is required?

Comment (chandrikamohith) [inline mifosng-provider/src/main/java/org/mifosplatform/portfolio/loanaccount/loanschedule/domain/DefaultScheduledDateGenerator.java]: Incrementing adjustedDate by 1 day if condition is true.Else we ll call the existing logic getOffSetDateIfNonWorkingDay.
